### PR TITLE
Add sample dashboard for GCE and GKE GPU Utilization.

### PR DESCRIPTION
**Pull request by sdlin**
This adds a new sample dashboard which combines GPU utilization for GCE and GKE.

GCE and GKE are separate resources, and so have two different metrics for GPU utilization (although they measure the same thing).  All graphs in this dashboard "union" the two metrics.

GCE Metric Docs (requires Ops Agent): https://cloud.google.com/monitoring/api/metrics_opsagent#agent-gpu

GKE Metric Docs: https://cloud.google.com/monitoring/api/metrics_kubernetes

### CODE REVIEW

**Review by yqlu — COMMENTED** [review]
Dashboard json, screenshots, metadata.yaml LGTM.

Can you update https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/nvidia-gpu/README.md in tandem too?

**Comment by sdlin** [comment]
> Dashboard json, screenshots, metadata.yaml LGTM.
> 
> Can you update https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/nvidia-gpu/README.md in tandem too?

Done!

File: dashboards/nvidia-gpu/README.md [hunk 1]
```diff
@@ -15,3 +16,8 @@
 |:------------------|
 |Filename: [nvidia-dcgm.json](nvidia-dcgm.json)|
 |This dashboard has charts for viewing NVIDIA GPUs when monitored by Google's GPU enabled Ops Agent configured to use the DataCenter GPU Manager. It displays GPU utilization, GPU used memory, active GPU processes, Streaming Multiprocessor (SM) block utilization, SM occupancy, SM pipe utilization, PCIe traffic, and NVLink traffic.|
+
+|GCE & GKE GPU Utilization|
+|:------------------|
+|Filename: [gce-gke-gpu-utilization.json](gce-gke-gpu-utilization.json)|
+|This dashboard combines GCE and GKE GPU counts and utitilization.|
```
Comment (yqlu) [inline dashboards/nvidia-gpu/README.md:23]: utitilization -> utilization :)

Can you sync this description with the metadata.yaml one? Not super critical for them to be exactly the same but we're in the middle of a pipeline migration that parses the README to one that will parse the metadata.yaml.

Comment (sdlin) [reply dashboards/nvidia-gpu/README.md]: Thanks!

Comment (sdlin) [reply dashboards/nvidia-gpu/README.md]: Let me also sync the description.

Comment (sdlin) [reply dashboards/nvidia-gpu/README.md]: Done.

**Review by yqlu — APPROVED** [review]

**Pull request merged**